CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 3.11)
+cmake_minimum_required (VERSION 3.16)
 
```

Comment:
Strictly speaking, you don't need to modify the version here. You don't use any features that require a higher version. SeqAn will already check for 3.16